berry_server/main.go

```diff
@@ -14,6 +14,8 @@ import (
 // server is used to implement hellowrld.GreeterServer.
 type server struct{}
 
+const grpcPort = "31337"
```

Comment:
we could use something like this :

var port int

func init() {
	flag.IntVar(&port, "port", 31337, "grpcs port")
}

and in main() {
flag.Parse()
.
.
.

but probably port will be hardcoded so it could stay as it is